superdesk/commands/data_updates.py

```diff
@@ -76,7 +76,7 @@ def get_data_updates_files(strip_file_extension=False):
     # create folder if doens't exist
     for folder in get_dirs():
         if not os.path.exists(folder):
-            os.makedirs(folder)
+            continue
```

Comment:
I think we need this folder to be created somewhere. Maybe putting a `try...except OSError: pass` around `os.makedirs(...)` could solve the initial problem ?